single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/builders/SubstationRawBuilder.java

```diff
@@ -53,9 +54,6 @@ public Map<VoltageLevelRawBuilder, FeederNode> createLine(String id, VoltageLeve
         if (containsVoltageLevelRawBuilder(vl1) && containsVoltageLevelRawBuilder(vl2)) {
             // All VoltageLevel must be in the same Substation
```

Comment:
Actually, lines can exist between substations (and are mainly between substations). Do we keep this condition because lines between substations will be dealt with at the zone graph level?

Reply:
Indeed without these conditions an NullPointerException will be raised as described in issue #534